src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+function sliding(e) {
+  const targetLi = e.target.closest('li');
+  const subList = targetLi.querySelector('ul');
+
+  if (targetLi && subList) {
+    subList.style.display = subList.style.display === 'none' ? '' : 'none';
```

Comment:
Consider setting the initial display state of the sub-lists to 'none' in the CSS or JavaScript to ensure consistent toggle behavior.